setup.cfg

```diff
@@ -1,10 +1,66 @@
 [metadata]
-package_name = jwst
-description = JWST Pipeline
-author = STScI
-license = BSD
-edit_on_github = False
-github_project = spacetelescope/jwst
+name = jwst
+description = Python library for science observations from the James Webb Space Telescope
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = JWST Pipeline developers
+license = BSD 3-Clause License
+url = https://github.com/spacetelescope/jwst
+classifiers =
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering :: Astronomy
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
+
+[options]
+zip_safe = False
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+install_requires =
+    asdf>=2.5
+    astropy>=4.0
+    crds>=7.4.1.3
+    drizzle>=1.13
+    gwcs>=0.13.0
+    jsonschema>=3.0.1
+    numpy>=1.16
```

Comment:
This looks pretty old too. Should we go with v 0.17 as the oldest? The current is 0.19.

Reply:
We actually test these in our "Oldest Dependencies" Travis CI job. I.e. `numpy 1.16` actually does work with all the other old dependencies.  So while people should be using newer versions, it is not required.  If it eventually stops working that Oldest Dependencies Travis job will start failing and we'll have to update this.

And that oldest dependencies Travis job actually gets the spec from `install_requires` minimums.

Reply:
Ah, I wasn't aware of that job. Awesome!

Reply:
Yes, there's a script that generates a `requirements-min.txt` file.  The Travis job runs it, installs those deps and runs the unit tests on that installation to verify they are correct.

https://github.com/spacetelescope/jwst/blob/master/scripts/minimum_deps

We also have a script that verifies that `install_requires` lists everything that the package actually needs at runtime.  I.e. if someone adds a runtime dependency on `pytest` or other test or doc dependencies, then this will catch it.

https://github.com/spacetelescope/jwst/blob/master/scripts/verify_install_requires

We had both the above things cause our `install_requires` to get out-of-date, and in the latter case, break a delivered tagged version of the code.